selfdrive/car/hyundai/values.py

```diff
@@ -248,7 +248,7 @@ def init_make(self, CP: car.CarParams):
   ],
   CAR.GENESIS_G70: HyundaiCarInfo("Genesis G70 2018-19", "All", harness=Harness.hyundai_f),
   CAR.GENESIS_G70_2020: HyundaiCarInfo("Genesis G70 2020", "All", harness=Harness.hyundai_f),
-  CAR.GENESIS_GV70_1ST_GEN: HyundaiCarInfo("Genesis GV70 2022-23", "All", harness=Harness.hyundai_l),
+  CAR.GENESIS_GV70_1ST_GEN: HyundaiCarInfo("Genesis GV70 2022-23", "All", harness=Harness.hyundai_m),
```

Comment:
Quickly searched Discord and didn't see any other users complain of this harness not working, so let's try to figure that out first before we change.